src/Chainweb/Pact/PactService.hs

```diff
@@ -581,9 +586,9 @@ validateChainwebTxs
     -> BlockHeight
     -> Vector ChainwebTransaction
     -> RunGas
-    -> IO (Vector Bool)
+    -> IO (Vector (Either Mempool.InsertError ()))
```

Comment:
Was there a reason this function is in `IO` as opposed to `PactServiceM` here? I grepped and it's only used in this file as part of the service flow.

Reply:
No, we can fix it later.